api/managementpb/dbaas/psmdb_cluster.proto

```diff
@@ -108,14 +108,6 @@ message GetPSMDBClusterResponse {
     // Replicaset name.
     string replicaset = 5;
   }
-  // Cluster state.
-  PSMDBClusterState state = 1;
-  // Currently running operation, if any.
-  RunningOperation operation = 2;
-  // Cluster parameters.
-  PSMDBClusterParams params = 3;
-  // Connection credentials.
-  PSMDBCredentials connection_credentials = 4;
```

Comment:
let's keep credentials :)

Reply:
They are here now, I just moved the PSMDBCredentials outside of the response message. Or did you mean keep the message definition inside the response message definition?

Reply:
Yeah, let's keep it inside since we don't use `PSMDBCredentials` anywhere else.

Reply:
done